app.js

```diff
@@ -2,6 +2,8 @@ const express = require('express');
 const app = express();
 const port = process.env.PORT || 3000;
 const multer = require('multer');
+const mongoose = require('mongoose');
+const Prof = require('./models/countryModel'); //import schema
```

Comment:
What is the name `Prof` referring to? Seeing the import it's clear It's a schema but the name isn't very descriptive.

Reply:
Maybe a different name is more clearly. But I've chosen Prof because of the profile collection:

The first argument is the singular name of the collection your model is for. Mongoose automatically looks for the plural, lowercased version of your model name. Thus, for the example above, the model Prof is for the profiles collection in the database. (https://mongoosejs.com/docs/models.html)

Maybe it is better to use a country collection to store the data from my function.